tfjs-converter/src/executor/model_analysis.ts

```diff
@@ -178,18 +175,16 @@ export function getNodeLiveUntilMap(orderedNodes: Node[]): Map<Node, Node[]> {
 
   for (let nodeOrder = 0; nodeOrder < nNodes; ++nodeOrder) {
     const node = orderedNodes[nodeOrder];
-    for (const child of node.children) {
-      const childOrder = nodeToOrder.get(child)!;
-      // Extend the node's life to at least its child's life.
-      liveUntil[nodeOrder] =
-          Math.max(liveUntil[nodeOrder], liveUntil[childOrder]);
-    }
+    // Extend the node's life to at least its child's life.
+    liveUntil[nodeOrder] = Math.max(
+        liveUntil[nodeOrder],
+        ...node.children.map((child) => liveUntil[nodeToOrder.get(child)!]));
```

Comment:
`Math.max` supports a limited number of arguments before running into `maximum call stack size exceeded`. In Node, I've experimentally found this limit to be around `123258`, which is rather small. I recommend using `reduce` here to avoid that limit.

```suggestion
    liveUntil[nodeOrder] = node.children.reduce((currentMax, child) => 
        Math.max(liveUntil[nodeToOrder.get(child)!], currentMax), liveUntil[nodeOrder]);
```